src/_InputValidation.tsx

```diff
@@ -45,7 +45,7 @@ const InputValidation: React.FC<React.PropsWithChildren<Props>> = ({children, id
       }}
     >
       {IconComponent && (
-        <Box as="span" mr={1} sx={{display: 'flex'}} aria-hidden="true">
+        <Box as="span" display="flex" mr={1} sx={{transform: 'translateY(2px)'}} aria-hidden="true">
```

Comment:
I had to deal with a similar problem (offsetting icon to align with the text). So I am wondering:
Why do you prefer `translate` vs margin?
Is it correct to assume that alignment will break if user changes the font or font size?

Reply:
Yes, the alignment would break if **we** changed the font size, but it doesn't break if the user zooms their browser in or out.